package.json

```
@@ -28,7 +28,8 @@
     "test": "jest && rimraf 'test/resources/**/execution-*'",
     "lint": "eslint '**/*.js' --ignore-pattern node_modules/",
     "lint:fix": "eslint '**/*.js' --ignore-pattern node_modules/ --fix",
-    "prepublish": "npm test"
+    "prepublish": "npm test",
+    "perf": "node test/perf/timing.js && rimraf 'test/resources/**/execution-*'"
```

sometimes it is better to be more verbose
```suggestion
    "test:performance": "node test/perf/timing.js && rimraf 'test/resources/**/execution-*'"
```

---

Ah gotcha will update it